packages/utils/src/instrument/fetch.ts

```diff
@@ -57,44 +74,102 @@ function instrumentFetch(): void {
 
       // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
       return originalFetch.apply(GLOBAL_OBJ, args).then(
-        (response: Response) => {
-          const finishedHandlerData: HandlerDataFetch = {
-            ...handlerData,
-            endTimestamp: timestampInSeconds() * 1000,
-            response,
-          };
-
-          triggerHandlers('fetch', finishedHandlerData);
+        async (response: Response) => {
+          if (handlerType === 'fetch-body-resolved') {
+            // clone response for awaiting stream
+            let clonedResponseForResolving: Response | undefined;
+            try {
+              clonedResponseForResolving = response.clone();
+            } catch (e) {
+              // noop
+              DEBUG_BUILD && logger.warn('Failed to clone response body.');
```

Comment:
l: I'd vote to leave this log away, it increases bundle size (a bit) and is not that critical, if that fails we don't really care, I'd say?

Reply:
Sounds good!